Dynatrace/dynatrace-operator
# Fix hostGroup conversion logic for v1beta2

## Description
Ticket: https://dt-rnd.atlassian.net/browse/K8S-10198

Currently the hostGroup parameter is not converted from v1beta1 to v1beta2.


## How can this be tested?
Install Operator -> Apply a v1beta1 Dynakube with spec.oneAgent.hostGroup -> check if the converted v1beta2 Dynakube also has hostGroup set
